migrations/20230512130000-add-comment-type.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const CommentType = {
+  COMMENT: 'COMMENT',
+  PRIVATE_NOTE: 'PRIVATE_NOTE',
+};
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.addColumn('Comments', 'type', {
+      type: Sequelize.ENUM(...Object.values(CommentType)),
+      defaultValue: CommentType.COMMENT,
+    });
+    await queryInterface.addColumn('CommentHistories', 'type', {
+      type: Sequelize.ENUM(...Object.values(CommentType)),
+      defaultValue: CommentType.COMMENT,
+    });
```

Comment:
If not meant to be nulled

```suggestion
    await queryInterface.addColumn('Comments', 'type', {
      type: Sequelize.ENUM(...Object.values(CommentType)),
      defaultValue: CommentType.COMMENT,
      allowNull: false
    });
    await queryInterface.addColumn('CommentHistories', 'type', {
      type: Sequelize.ENUM(...Object.values(CommentType)),
      defaultValue: CommentType.COMMENT,
      allowNull: false
    });
```